deployment/ansible/roles.yml

```diff
@@ -8,4 +8,4 @@
   version: 1.0.1
 
 - src: azavea.nginx
-  version: 0.3.0
\ No newline at end of file
+  version: 0.3.0
```

Comment:
Good catch. Addressed in https://github.com/azavea/raster-foundry/pull/430/commits/9e51d281e67440f24ba6a15539f15d91a7312c43.